.github/workflows/check-examples.yml

```diff
@@ -40,9 +40,13 @@ jobs:
         python-version: 3.8
     - name: install python requirements for notebooks
       run: |
+        # if version are updated, update the version in the build-manual file
+        # and the requirements.txt, too.
+        # this duplication is currently necessary because of an issue with the pyjwt
+        # version required from qcs-api-client < 0.21.0
```

Comment:
This comment doesn't need to go here (since this file doesn't need updating).

Reply:
Oh yes, forgot to update. Changed now.